testing/tests/DevExpress.ui.widgets.form/form.API.option_formData.tests.js

```diff
@@ -329,3 +329,118 @@ QUnit.test('Reset editor\'s value when set formData: {dataField1: a}', function(
         assert.deepEqual(editor.option('value'), editor._getDefaultOptions().value, `a default value of the ${dataField} editor`);
     });
 });
+
+
+QUnit.module('Checkbox editor field', () => {
+    function checkCheckboxAndFormValue(form, dateField, editorValue, formValue) {
+        const editor = form.getEditor(dateField);
+        QUnit.assert.strictEqual(editor.option('value'), editorValue, `editor has ${editorValue} value`);
+
+        const expectedFormValue = formValue === 'no member' ? undefined : formValue;
+        QUnit.assert.strictEqual(form.option(`formData.${dateField}`), expectedFormValue, `formData has ${expectedFormValue} value`);
```

Comment:
QUnit.assert.strictEqual(form.option(`formData.${dateField}`), formValue === 'no member' ? undefined : formValue, 'expected form value'); - instead of "const expectedFormValue" ?

Reply:
There is also comment `formData has ${expectedFormValue} value`
I think with this comment it’s easier to understand which check has fail